### ISSUE TITLE
GPU missing key error in Cascaded architecture tutorial

### ISSUE THREAD

**Issue by mariehbourget** [ivadomed/ivadomed#709]
While running the tutorial `Cascaded architecture`, I have this error:
```
Traceback (most recent call last):
  File "/home/mhbourget/venv-ivadomed-38c/bin/ivadomed", line 11, in <module>
    load_entry_point('ivadomed', 'console_scripts', 'ivadomed')()
  File "/home/mhbourget/ivadomed/ivadomed/main.py", line 519, in run_main
    run_command(context=context,
  File "/home/mhbourget/ivadomed/ivadomed/main.py", line 347, in run_command
    ds_valid = get_dataset(loader_params, valid_lst, transform_valid_params, cuda_available, device, 'validation')
  File "/home/mhbourget/ivadomed/ivadomed/main.py", line 113, in get_dataset
    ds = imed_loader.load_dataset(**{**loader_params,
  File "/home/mhbourget/ivadomed/ivadomed/loader/loader.py", line 93, in load_dataset
    dataset = BidsDataset(path_data,
  File "/home/mhbourget/ivadomed/ivadomed/loader/loader.py", line 827, in __init__
    bounding_box_dict = imed_obj_detect.load_bounding_boxes(object_detection_params,
  File "/home/mhbourget/ivadomed/ivadomed/object_detection/utils.py", line 254, in load_bounding_boxes
    object_detection_params['gpu'],
KeyError: 'gpu'
```
The key `gpu` is missing in `object_detection_params` in the `config.json` used in the tutorial.
The key should be add to `config_default.json`, or a default value set in [load_bounding_box](https://github.com/ivadomed/ivadomed/blob/321a91c7e3c82e6296764895e39695b04a80c8af/ivadomed/object_detection/utils.py#L254).

**Comment by andreanne-lemay** [comment]
Hum this is weird, it should be added when processing the file in the main: https://github.com/ivadomed/ivadomed/blob/master/ivadomed/main.py#L175. Maybe in the tutorial, the code doesn't enter this condition for some reason? Should be investigated! But we shouldn't need to add this parameter to the config file.

**Comment by mariehbourget** [comment]
> Hum this is weird, it should be added when processing the file in the main

Good to know, thanks!
I see the issue now: the key is `gpu_ids` in `main`, but `gpu` in `load_bounding_box`.

**Comment by andreanne-lemay** [comment]
Ah yes make sense! Easy to fix then! This key was not changed when the "gpu" key changed name.

**Comment by mariehbourget** [comment]
Should I fix it now in #699, as I'm replacing `object_detection/utils_new.py` by `utils.py`?

**Comment by andreanne-lemay** [comment]
Yes, since it's an easy fix I think we can include it in your PR! Thanks for noticing / fixing this :D
